tags.go

```diff
@@ -0,0 +1,31 @@
+package phpserialize
+
+import "strings"
+
+type tagOptions string
+
+func parseTag(tag string) (string, tagOptions) {
+	if i := strings.Index(tag, ","); i != -1 {
+		return tag[:i], tagOptions(tag[i+1:])
+	}
+	return tag, ""
+}
+
+func (o tagOptions) Contains(option string) bool {
+	if len(o) == 0 {
```

Comment:
I'm curious why this needs to be so complex. Couldn't you just do this?

```
func (o tagOptions) Contains(lookingFor string) bool {
    options := strings.Split(o, ",")
    for _, option := range options {
        if option == lookingFor {
            return true
        }
    }
    return false
}
```

Reply:
It's the logic used for tag parsing in the `json` package. I tried a function like yours at first too, but the other one was around 9x faster (benchmark showed 28ns/op vs 250ns/op), so I just went with that one.

Reply:
OK, fair enough.